### PR TITLE
Integrate EditorConfig

### PR DESCRIPTION
**Pull request by danielpetisme**
[EditorConfig](http://editorconfig.org/) is a neat tool to enhance coding styles. 
EditorConfig integration is almost free (a single dotfiles) and widely supported (IntelliJ, Atom, Sublime, Vim, etc.)

Thoughts?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -39,7 +39,7 @@ ext {
 // .................................................................................................................. //
 
 group = 'org.eclipse.golo'
-version = '3.1.0'
```
Comment (yloiseau) [inline build.gradle]: oops? :smile:

Comment (danielpetisme) [inline build.gradle]: A typo from previous changes, ping @jponge.

Comment (jponge) [inline build.gradle]: Yep, just bump it to `3.1.1-SNAPSHOT`

Comment (jponge) [inline build.gradle]: No it should be `3.1.1-SNAPSHOT` on that branch :wink:

File: golo-maven-plugin/pom.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
   <groupId>org.eclipse.golo</groupId>
   <artifactId>golo-maven-plugin</artifactId>
-  <version>3.1.0</version>
+  <version>3.2.0-SNAPSHOT</version>
```
Comment (jponge) [inline golo-maven-plugin/pom.xml]: Same here

**Comment by jponge** [comment]
Ok I see why there's a conflict on the Gradle & Maven build descriptors: you somehow brought a commit from me into that branch.

I'll manually pick this PR and turn it into a clean single commit on the the right branch.

**Comment by jponge** [comment]
:white_check_mark:

**Comment by danielpetisme** [comment]
Thanks, Indeed I noticed it too but not time to fix it.... :+1:

**Pull request closed without merging**